controller/src/beerocks/master/son_master_thread.cpp

```diff
@@ -3574,7 +3601,7 @@ bool master_thread::handle_cmdu_control_message(const std::string &src_mac,
         new_event.radio_mac = radio_mac;
         auto taskID         = database.get_dynamic_channel_selection_task_id(radio_mac);
         if (taskID == -1) {
-            LOG(ERROR) << "no task for the requesed mac (" << radio_mac << ") could be found!";
+            LOG(ERROR) << "no task for the requested mac (" << radio_mac << ") could be found!";
```

Comment:
```suggestion
            LOG(ERROR) << "No DCS task for the requested mac (" << radio_mac << ") is currently running!";
```

Reply:
not part of the changes done - but fixed anyways :-)